src/Form/RegisterType.php

```diff
@@ -17,12 +17,7 @@ class RegisterType extends AbstractType
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-            ->add('username', TextType::class, [
-                'constraints' => new Length([
```

Comment:
why did you remove this constraint? I can pass now a very long string and it will fail when database insert will happen
By the way - try to move your validation checks to forms

Reply:
Redo